UltiSnips/javascript_jasmine.snippets

```diff
@@ -1,10 +1,6 @@
 priority -50
 
-# JavaScript versions -- from the TextMate bundle + some additions
-# for jasmine-jquery matchers
-#
-
-snippet des "Describe (js)" b
+snippet desc "Describe (js)" b
```

Comment:
`desc` is the trigger used in other popular BDD frameworks, I think it's probably better to keep things consistent, as recommended in the readme.